### ISSUE TITLE
After clean I can't run meteor project

### ISSUE THREAD

**Issue by thearabbit** [klaussner/meteor-cleaner#4]
After I `meteor clean-package-cache --keep-scanned`, I can't run meteor project.
```js
// In my project
meteor
```
Get error
```js
➜  pos-vue git:(master) ✗ meteor
'/Users/theara/.meteor' exists, but '/Users/theara/.meteor/meteor' is not executable.

Remove it and try again.
```

**Comment by klaussner** [comment]
Sorry about that! I think the problem is that `~/.meteor/meteor` is a symlink to the latest version of the `meteor-tool` package (in the `packages` folder). If that version is removed, the symlink doesn't point to an existing executable anymore. Could you post the output of the following two commands?

```
ls -al ~/.meteor
ls -al ~/.meteor/packages/meteor-tool
```

**Comment by klaussner** [comment]
The bug is fixed in version 1.0.1 but you probably have to reinstall Meteor with `curl https://install.meteor.com/ | sh` to repair the symlink. However, if there still is a version of the `meteor-tool` package in `~/.meteor/packages/meteor-tool`, you could just fix the symlink instead.
